content/fr/agent/basic_agent_usage/windows.md

```diff
@@ -69,32 +69,34 @@ Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-agent-7-latest.amd64.m
-| `PROXY_HOST`               | Chaîne | En cas d'utilisation d'un proxy, définit le host de votre proxy. [En savoir plus sur l'utilisation d'un proxy avec l'Agent Datadog][3].                                                                                                                                 |
-| `PROXY_PORT`               | Nombre | En cas d'utilisation d'un proxy, définit le port de votre proxy. [En savoir plus sur l'utilisation d'un proxy avec l'Agent Datadog][3].                                                                                                                                 |
-| `PROXY_USER`               | Chaîne | En cas d'utilisation d'un proxy, définit l'utilisateur de votre proxy. [En savoir plus sur l'utilisation d'un proxy avec l'Agent Datadog][3].                                                                                                                                 |
-| `PROXY_PASSWORD`           | Chaîne | En cas d'utilisation d'un proxy, définit le mot de passe de votre proxy. Pour l'Agent de processus ou de conteneur, cette variable est requise pour la transmission d'un mot de passe d'authentification. Elle ne peut pas être renommée. [En savoir plus sur l'utilisation d'un proxy avec l'Agent Datadog][3]. |
-| `DDAGENTUSER_NAME`         | Chaîne | Remplace le nom d'utilisateur `ddagentuser` par défaut lors de l'installation de l'Agent _(version 6.11.0+)_. [En savoir plus sur l'utilisateur de l'Agent Windows Datadog][2].                                                                                      |
-| `DDAGENTUSER_PASSWORD`     | Chaîne | Remplace le mot de passe chiffré généré pour l'utilisateur `ddagentuser` lors de l'installation de l'Agent _(version 6.11.0+)_. Doit être spécifié pour les installations sur les serveurs DNS. [En savoir plus sur l'utilisateur de l'Agent Windows Datadog][2].  |
-| `APPLICATIONDATADIRECTORY` | Chemin   | Remplace le répertoire à utiliser pour l'arborescence du fichier de configuration. Peut uniquement être spécifié à l'installation initiale, pas lors des mises à jour. Valeur par défaut : `C:\ProgramData\Datadog`. _(version 6.11.0+)_                                           |
-| `PROJECTLOCATION`          | Chemin   | Remplace le répertoire à utiliser pour l'arborescence du binaire. Peut uniquement être spécifié à l'installation initiale, pas lors des mises à jour. Valeur par défaut : `%PROGRAMFILES%\Datadog\Datadog Agent`. _(v6.11.0+)_                                    |
+| Variable                                    | Type    | Description                                                                                                                                                                                                                         |
```

Comment:
@davidejones Hey there! I checked this in the preview, but it looks like it's not showing up in the staging doc: https://docs-staging.datadoghq.com/guacbot/translation-20211203082803/fr/agent/basic_agent_usage/windows/?tab=agentsv6etv7#configuration

Reply:
It looks like something is messing up the tabs here I'll try take a closer look

Reply:
@sarina-dd I added a minor fix commit for a broken link but other than that i see this in the preview it was just under another tab see this link https://docs-staging.datadoghq.com/guacbot/translation-20211203082803/fr/agent/basic_agent_usage/windows/?tab=lignedecommande#configuration